package.json

```
@@ -1965,6 +1965,46 @@
 				},
 				"description": "The color used for indicating that an issue is closed."
 			},
+			{
+				"id": "pullRequests.merged",
+				"defaults": {
+					"dark": "#8957e5",
+					"light": "#8957e5",
+					"highContrast": "editor.background",
+					"highContrastLight": "editor.background"
+				},
+				"description": "The color used for indicating that a pull request is merged."
+			},
+			{
+				"id": "pullRequests.draft",
+				"defaults": {
+					"dark": "#6e7681",
+					"light": "#6e7681",
+					"highContrast": "editor.background",
+					"highContrastLight": "editor.background"
+				},
+				"description": "The color used for indicating that a pull request is a draft."
+			},
+			{
+				"id": "pullRequests.open",
+				"defaults": {
+					"dark": "#238636",
+					"light": "#238636",
```

Let's base this off of the `issues.open` color so we don't have two slightly different open colors.
```suggestion
					"dark": "issues.open",
					"light": "issues.open",
```